Source/Bogus/Faker[T].cs

```diff
@@ -235,8 +235,8 @@ public virtual Faker<T> Rules(Action<Faker, T> setActions)
                return null;
             };
          var guid = Guid.NewGuid().ToString();
-         var rule = new PopulateAction<T>()
             {
+         var rule = new PopulateAction<T>
```

Comment:
Looks like an incorrect syntax here again. The statements look like they get moved below the object initializer brace.